File: Easy/2026-02-16-190-Reverse-Bits/solution.java
public class Solution {
    // Function to reverse bits of a given 32 bits signed integer
    public int reverseBits(int n) {
        int result = 0;
        for (int i = 0; i < 32; i++) {
            result <<= 1; // Shift result to the left
            result |= (n & 1); // Add the last bit of n to result
            n >>= 1; // Shift n to the right
        }
        return result;
    }
}

File: Easy/2026-02-16-190-Reverse-Bits/solution.js
var reverseBits = function(n) {
    let reversed = 0;
    for (let i = 0; i < 32; i++) {
        reversed <<= 1;
        reversed |= (n & 1);
        n >>= 1;
    }
    return reversed >>> 0; // Ensure the result is treated as an unsigned 32-bit integer
};